src/agoradatatools/etl/transform/immunohisto_transform.py

```diff
@@ -258,28 +332,31 @@ def immunohisto_transform(
 
     Args:
         datasets (Dict[str, pd.DataFrame]): Dictionary of dataset names mapped to their DataFrame.
+            Can optionally include 'immunohisto_measure_order' key with ordering configuration.
```

Comment:
This doesn't actually seem to be optional. Your code throws an error if the measure order dataset doesn't exist.